setup.py

```diff
@@ -6,12 +6,12 @@
     install_requires=[
         "dagster>=1.0.0",
         "dagster_airflow>=0.17.5",
-        "apache-airflow==2.5.0",
+        "apache-airflow==2.5.2",
         "dagster_cloud>=1.0.0",
         # airflow dag specific dependencies
         "docker>=5.0.3,<6.0.0",
         "apache-airflow-providers-docker>=3.2.0,<4",
-        "apache-airflow-providers-apache-spark>=3.0.0,<4",
+        "apache-airflow-providers-apache-spark>=4.0.1",
         "kubernetes>=10.0.1",
```

Comment:
do we actually need any of these for this example?

Reply:
yes the example dags import these